static/js/components/LearnerSearch.js

```diff
@@ -142,6 +173,7 @@ export default class LearnerSearch extends SearchkitComponent {
                     role="button"
                     id="email-selected"
                     className="mm-button"
+                    onClick={this.openEmailComposer}
```

Comment:
ah yeah we might be able to do something like that, I was trying to get away from re-declaring the function on every re-render. Before I was passing in a function that took `searchkit` and an argument and then returns an `onClick` handler, but I realized it was being called constantly, which was creating some strange effects with a variable I was trying to keep in closure (the current query object at the time of opening the dialog).

I'll give that a try!